Source/WebCore/page/Page.cpp

```diff
@@ -3888,6 +3889,13 @@ void Page::abortApplePayAMSUISession(ApplePayAMSUIPaymentHandler& paymentHandler
 
 #endif // ENABLE(APPLE_PAY_AMS_UI)
 
+#if ENABLE(MEDIA_SOURCE)
+void Page::enableMockMediaSource()
+{
+    platformStrategies()->mediaStrategy().enableMockMediaSource();
```

Comment:
It could be a free function or a static function.
Not sure this is worth it.

Reply:
agreed, I've removed it